packages/core/test/financial-templates/common/FundingRateApplier.js

```diff
@@ -30,55 +30,54 @@ contract("FundingRateApplier", function() {
     let fundingRateApplier = await FundingRateApplier.new(fpFinder.address, fundingRateIdentifier, timer.address);
 
     assert.equal(await fundingRateApplier.cumulativeFundingRateMultiplier(), toWei("1"));
-    assert.equal(await fundingRateApplier.fpFinder(), fpFinder.address);
   });
 
   it("Computation of effective funding rate and its effect on the cumulative multiplier is correct", async () => {
     let fundingRateApplier = await FundingRateApplier.new(fpFinder.address, fundingRateIdentifier, timer.address);
 
     // Funding rate of 0.15% charged over 20 seconds on a starting multiplier of 1:
-    // Effective Rate: 0.0015 * 20 = 0.03, funding rate is positive so add +1 => 1.03
+    // Effective Rate: 0.0015 * 20 = 0.03, funding rate is positive so add 1 => 1.03
     // Cumulative Multiplier: 1 * 1.03 = 1.03
     const test1 = await fundingRateApplier.calculateEffectiveFundingRate(
       20,
-      { rawValue: toWei("1.0015") },
```

Comment:
You need to do the same update to `FundingRateStore` in `PerpetualLiquidatable.js` and `PerpetualPositionManager.js` to accommodate centering around 0 as opposed to 1 how it is now. CI is failing on 10 tests as a result of this.

Reply:
Done. These should all be fixed now.